docs/src/components/productX/XGridFullDemo.tsx

```diff
@@ -177,7 +178,7 @@ export default function XGridFullDemo() {
             {
               overflow: 'auto',
```

Comment:
@cherniavskii What about the following change to remove the scroll bar on the right?

```suggestion
              overflow: 'hidden',
```

![image](https://github.com/mui/material-ui/assets/45398769/1df7df61-4e8b-4179-8c47-4f2e933985af)